src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -362,21 +323,10 @@ public static double get(final double difference) {
         }
 
         private static void initialize() {
-            if (cache == null) {
-                final int tableSize = (int) (MAX_TOLERANCE / TABLE_STEP) + 1;
-                cache = new double[tableSize];
-                for (int k = 0; k < cache.length; k++) {
-                    cache[k] = Math.log10(1.0 + Math.pow(10.0, -((double) k) * TABLE_STEP));
-                }
-            }
+            final int tableSize = (int) (MAX_TOLERANCE / TABLE_STEP) + 1;
```

Comment:
i think we should put back the null check and make the initialize method synchronized to prevent race conditions in case this cache is accessed via multiple threads
(the null check is needed because another thread may have created the cache after null is checked in `get`)